geomesa-convert/geomesa-convert-common/src/test/scala/org/locationtech/geomesa/convert2/TypeInferenceTest.scala

```diff
@@ -47,12 +47,14 @@ class TypeInferenceTest extends Specification with LazyLogging {
 
   "TypeInference" should {
     "infer simple types" in {
-      val types = TypeInference.infer(Seq(Seq("a", 1, 1L, 1f, 1d, true)))
+      // note: don't put any valid lat/lon pairs next to each other or it will create a geometry type
```

Comment:
Uhhmmm...  so yeah, type inference on data with lots of 'smallish' numeric fields could go poorly?

Reply:
not really, just for this test I didn't want to get any inferred geometries. if there are numeric fields, it will infer the first 2 as lon/lat for a geometry type (assuming no other geometry field exists). we don't really have any way to distinguish between fields at that point, so the user would have to modify the converter to get the right ones.

Reply:
also, if the fields are labelled (e.g from a csv header) as 'lat/lon' or 'latitude/longitude' we'll use those. I added a test to show that as well